File: Pset 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        // ask user for size
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);

    for (int i = n; i > 0; i--)
    {
        //
        for (int j = 1; j < i; j++)
        {
            printf(" ");
        }
        for (int k = -1; k < n - i; k++)
        {
            printf("#");
        }
        //
        printf("\n");
    }
}

File: Pset 6/mario.py
from cs50 import get_int
# repeat while input is invalid
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
# draw pyramid
i = height
while i > 0:
    for j in range(i - 1):
        print(" ", end="")
    for k in range(i - (i - 1)):
        for l in range(height - (i - 1)):
            print("#", end="")
        print()
    i = i - 1
